lua/todo-comments/highlight.lua

```diff
@@ -87,7 +87,7 @@ function M.highlight(buf, first, last, _event)
   if not vim.api.nvim_buf_is_valid(buf) then
     return
   end
-  vim.api.nvim_buf_clear_namespace(buf, Config.ns, first, last + 1)
+  vim.api.nvim_buf_clear_namespace(buf or 0, Config.ns, first, last + 1)
```

Comment:
The `or 0` is probably not required here. The `buf` is taken from `nvim_win_get_buf` which should always result in a number, and the `nvim_buf_is_valid` call should prevent invalid usage of it.

Reply:
Fixed in 24c0877.